lib/pages/people/view/person_view.dart

```diff
@@ -1,104 +1,19 @@
 import 'package:acs_upb_mobile/pages/people/model/person.dart';
 import 'package:acs_upb_mobile/pages/people/service/person_provider.dart';
-import 'package:acs_upb_mobile/resources/utils.dart';
-import 'package:acs_upb_mobile/widgets/icon_text.dart';
-import 'package:cached_network_image/cached_network_image.dart';
+import 'package:acs_upb_mobile/pages/people/view/courses_list_view.dart';
+import 'package:acs_upb_mobile/pages/people/view/person_banner.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_feather_icons/flutter_feather_icons.dart';
 import 'package:provider/provider.dart';
 
-class PersonView extends StatelessWidget {
-  const PersonView({Key key, this.person}) : super(key: key);
+import 'contact_info.dart';
 
+class PersonView extends StatefulWidget {
```

Comment:
This doesn't need to be stateful. The scrollbar controller can simply be a final variable.

Reply:
This class can be stateless. 
I'd still argue that a future `WebScrollBar` widget I mentioned earlier should be stateless. (for line indexed links or actions)

Reply:
Well yeah, that's what I was saying, that it should be stateless.